fix: remove fuse configmap and add note (#2965)
codenvy/che-docs: modules/administration-guide/pages/enabling-fuse-for-all-workspaces.adoc

For OpenShift versions before 4.15, the `io.openshift.podman-fuse: ""` annotation is also required.
====

+
[NOTE]
====
The Universal Development Image (UDI) includes the following logic in the entrypoint script to detect fuse-overlayfs and set the storage driver.

> If using a custom image, equivalent logic should be added to the image's entrypoint.

Reviewer: ```suggestion
If you use a custom image, you should add an equivalent logic the image's entrypoint.
```
